FindPOLine/OrderLinesFilters.js

```diff
@@ -34,7 +32,14 @@ import { getDateRangeValueAsString } from './utils';
 
 const applyFiltersAdapter = (applyFilters) => ({ name, values }) => applyFilters(name, values);
 
-export function OrderLinesFilters({ activeFilters, applyFilters, disabled, funds = [], materialTypes = [] }) {
+export function OrderLinesFilters({
+  activeFilters,
+  applyFilters,
+  crossTenant = false,
+  disabled = false,
```

Comment:
move to `OrderLinesFilters.defaultProps`

Reply:
https://react.dev/blog/2024/04/25/react-19-upgrade-guide#removed-proptypes-and-defaultprops